app/ui-react/packages/ui/src/Data/Virtualizations/Views/ViewEditContent.tsx

```diff
@@ -1,9 +1,21 @@
+import 'codemirror/lib/codemirror.css';
```

Comment:
Yup, as @riccardo-forina said we should move all these code-mirror specific imports/requires into the `TextEditor` component.

Reply:
moved into TextEditor and amended the PR